tests/Data/ValidMembershipApplication.php

```diff
@@ -206,56 +220,72 @@ private function newAddress(): ApplicantAddress {
 	}
 
 	private function newPayment(): Payment {
-		return new Payment(
-			self::PAYMENT_PERIOD_IN_MONTHS,
-			Euro::newFromFloat( self::PAYMENT_AMOUNT_IN_EURO ),
-			$this->newDirectDebitPayment( $this->newBankData() )
+		return DirectDebitPayment::create(
```

Comment:
It might make sense to move the example payments to a different file, like we do in https://github.com/wmde/fundraising-donations/pull/177